java/pom.xml

```diff
@@ -8,10 +8,10 @@
     <groupId>org.seleniumhq.selenium.fluent</groupId>
     <artifactId>fluent-selenium</artifactId>
     <packaging>jar</packaging>
-    <version>1.18-SNAPSHOT</version>
+    <version>1.18-GUAVUS</version>
     <name>fluent-selenium</name>
     <description>A Fluent API on top of Selenium's WebDriver API</description>
-    <url>http://github.com/SeleniumHQ/fluent-selenium</url>
+    <url>https://github.com/Guavus/fluent-selenium</url>
```

Comment:
I have forked your project. You don't need to do anything, I pulled request against your project mistakenly.